go.mod

```diff
@@ -150,3 +148,5 @@ replace go.etcd.io/bbolt => go.etcd.io/bbolt v1.3.5
 
 // https://github.com/ory/dockertest/issues/212
 replace golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
+
+replace github.com/uber-go/tally => github.com/chronosphereio/tally v1.1.1-0.20210212113941-8eb2df32840b
```

Comment:
This is a draft PR in part because there's also a PR in progress for the tally update here.